external/README.md

```diff
@@ -1,3 +1,27 @@
 # External Dependencies
 
-TODO
+## Cap'n Proto v0.5.2
+
+**Note**: Compiling on newer versions of linux with glibc >=2.14 can be
+problematic. A change to memcpy causes linking to the newer glibc version,
+resulting in binaries that cannot be used on older platforms. See details
+and a potential workaround here:
+http://www.win.tue.nl/~aeb/linux/misc/gcc-semibug.html
+
+Some compiler flags are required and can be specified as below with the
+`CXXFLAGS` environment variable.
+
+```
+curl -O https://capnproto.org/capnproto-c++-0.5.2.tar.gz
+tar zxf capnproto-c++-0.5.2.tar.gz
+cd capnproto-c++-0.5.2
+CXXFLAGS="-fPIC -std=c++11 -m64 -fvisibility=hidden -Wall -Wreturn-type -Wunused -Wno-unused-parameter" ./configure
+make -j3 check
```

Comment:
Is this happening on the line you put the comment on? Instead of "make -j3 check" try just doing "make -j3" -- the check is optional and there are a number of cases where I have seen it erroneously fail (and even been encouraged to ignore it by the author).